models/generators.py

```diff
@@ -76,57 +76,67 @@ def __init__(self, img_size, num_latent_dims, kernel_size, num_filters,
                 lr : float
         """
         super().__init__()
+
+        self.w_init = tf.random_normal_initializer(stddev=0.02)
+        self.bn_init = tf.random_normal_initializer(1., 0.02)
```

Comment:
We probably need to find out what the best initialization for GANs are if this isn't from the paper

Reply:
Taken from the paper, too